### PR TITLE
Added some integration tests, work in progress

### PR DESCRIPTION
**Pull request by RokasSakys**

### CODE REVIEW

File: tests/IntegrationTests/RecipeDetailsTest.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+            HttpRequestMessage request = new HttpRequestMessage();
+            request.Method = HttpMethod.Delete;
+            request.RequestUri = uri;
+
+            var data = new[] { new KeyValuePair<String, String>("id", "3a577e39-5758-4de5-b3a1-3000a9a6db1f") };
+            request.Content = new FormUrlEncodedContent(data);
+
+
+            // Act
```
Comment (MazIeva) [inline tests/IntegrationTests/RecipeDetailsTest.cs:80]: clear commenting, nice!

File: vertinimai.md [hunk 2]
```diff
@@ -63,22 +63,3 @@
 | i | 2 | 2 | 2 | 2 |
 | d | 3 | 2 | 4 | 3 |
 11.23
-
```
Comment (MazIeva) [inline vertinimai.md:66]: Don't delete this!! I had just fixed it to have the latest marks!!

monkas

**Review by MazIeva — COMMENTED** [review]
Everything else looks good! Just fix it so that the latest marks don't get deleted!!

**Comment by popkovaite** [comment]
where are major problem because you overwrite my edit function and over changes:( so i think this need to be fix

**Pull request closed without merging**